### PR TITLE
[ISSUE #3919] fix postgres init sql has error 

### PR DESCRIPTION
**Pull request by MiracleDx**
About #3919 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
@MiracleDx hi，pls resolve conflicts

File: db/init/pg/create-table.sql [hunk 1]
```diff
@@ -1582,9 +1582,9 @@ INSERT INTO "public"."shenyu_dict" VALUES ('1529403902800302093', 'table', 'INIT
...
-INSERT INTO "public".`shenyu_dict` VALUES ('1545811989312315392', 'apidoc', 'API_DOC_GLOBAL_FLAG', 'status', 'true', 'Global switching (on or off) of API documents.', 0, 1, '2022-07-10 00:47:52', '2022-07-10 00:47:52');
-INSERT INTO "public".`shenyu_dict` VALUES ('1545812101056962560', 'apidocEnv', 'ENV_LABEL_OFFLINE', 'Offline', 'http://127.0.0.1:9195', 'Offline environment', 0, 1, '2022-07-10 00:48:19', '2022-07-10 00:48:19');
-INSERT INTO "public".`shenyu_dict` VALUES ('1545812228228259840', 'apidocEnv', 'ENV_LABEL_ONLINE', 'Online', 'http://127.0.0.1:9196', 'Online environment', 1, 1, '2022-07-10 00:48:49', '2022-07-10 00:48:49');
+INSERT INTO "public"."shenyu_dict" VALUES ('1545811989312315392', 'apidoc', 'API_DOC_GLOBAL_FLAG', 'status', 'true', 'Global switching (on or off) of API documents.', 0, 1, '2022-07-10 00:47:52', '2022-07-10 00:47:52');
+INSERT INTO "public"."shenyu_dict" VALUES ('1545812101056962560', 'apidocEnv', 'ENV_LABEL_OFFLINE', 'Offline', 'http://127.0.0.1:9195', 'Offline environment', 0, 1, '2022-07-10 00:48:19', '2022-07-10 00:48:19');
```
Comment (yu199195) [inline db/init/pg/create-table.sql:1586]: I think this pr only modify this.

File: db/init/pg/create-table.sql [hunk 2]
```diff
@@ -1604,11 +1604,11 @@ INSERT INTO "public"."shenyu_dict" VALUES ('1529402613195784281', 'saslMechanism
 -- ----------------------------
 DROP TABLE IF EXISTS "public"."user_role";
 CREATE TABLE "public"."user_role" (
-  "id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "user_id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "role_id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
-  "date_created" timestamp(6) NOT NULL DEFAULT timezone('UTC-8'::text, (now())::timestamp(0) without time zone),
-  "date_updated" timestamp(6) NOT NULL DEFAULT timezone('UTC-8'::text, (now())::timestamp(0) without time zone)
+                                      "id" varchar(128) COLLATE "pg_catalog"."default" NOT NULL,
```
Comment (yu199195) [inline db/init/pg/create-table.sql:1607]: not modify this. why format this sql?

Comment (MiracleDx) [reply db/init/pg/create-table.sql:1607]: maybe occured by github reslove conflict, my local file is alright

Comment (MiracleDx) [reply db/init/pg/create-table.sql:1607]: pls refused this pr

Comment (yu199195) [reply db/init/pg/create-table.sql:1607]: Hi， 
1.you close this pr. 
2.pull remote master,
3. modify only the places you modified.
4. privoder new pull reuqest

**Pull request closed without merging**